styles/cp-editor.css

```diff
@@ -163,6 +162,11 @@
   content: "\e990";
 }
 
+.h5p-course-presentation .h5p-slidecontrols-button.h5p-slidecontrols-button-aspect-ratio:before {
```

Comment:
Should we add a TODO statement for changing this to a more general solution when we have sorted out font icons?

Reply:
No need. This icon is likely to move quite soon anyway